ts-scripts/generateExternalTokens.ts

```diff
@@ -111,16 +112,23 @@ const formatApiTokenMetadata = async (
 
     if (denom.startsWith('ibc/')) {
       const ibcToken = await fetchIbcTokenMetaData(denom, Network.MainnetSentry)
+      const denomFromBank = getSupplyDenom(
+        denom,
+        Network.MainnetSentry
+      )
 
       if (ibcToken) {
         externalTokens.push({
           ...ibcToken,
+          ...(denomFromBank && { address: denomFromBank }),
```

Comment:
`address` and `denom` should be uppercase, like chain format. if we store them as lowercase in the `mainnet.json` files, you cant find the tokens as easily at product level, without situationally doing condional checks of `.toLowerCase`